src/App.js

```diff
@@ -1,25 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import BookList from './components/booksList/booksList';
+import BookForm from './components/booksForm/booksForm';
 
 function App() {
```

Comment:
## Employ the use of ES6 syntax
- [ ] Do you mind implementing this functional declaration using the `ES6` arrow syntax?